test/e2e/shoot/create_verify-adminKubeconfig_delete.go

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2022 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
Instead of creating a new shoot, should we simply make the name of the default test static and add the adminkubeconfig here? This way we would save some time/resources

Reply:
Can be done.

Reply:
But I was thinking of making this test run only on the prow pod, and skip it locally, since we need to alter the hosts file.

Reply:
Changing the `/etc/hosts` file is anyways needed for accessing the shoots, even locally. Either the `provider-local` extensions runs as a process (then it changes it right away) or it runs inside the kind cluster as container (then the user has to do it manually).

Reply:
Yes, got that.
But previously we didn't use to access the shoots externally during tests, so we didn't need sudo access.
If we skip this locally, then users can run other tests like before. That's what I meant.

Reply:
Personally, I would still vote to use the default create/delete test and extend it with the admin kubeconfig. I think it's an unnecessary overkill to create a separate shoot just for this (there is no different Shoot configuration whatsoever), so I think we can safe the resources and time. The "price" seems acceptable to me.